src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const arr = [];
+  const useState = { ...state };
```

Comment:
```suggestion
  const stateCopy = { ...state };
```
or stateClone, what does `useState` mean?

Reply:
Это я создал переменную что пришло первое в голову что бы проверить будет ли работать, а потом забыл переименовать. Ваша переменная логичнее.